gate-web/src/main/groovy/com/netflix/spinnaker/gate/ratelimit/RateLimitingInterceptor.java

```diff
@@ -81,7 +83,8 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
     RateLimitPrincipal principal =
         rateLimitPrincipalProvider.getPrincipal(principalName, sourceApp);
 
-    Rate rate = rateLimiter.incrementAndGetRate(principal);
+    final String handlerMethod = (handler instanceof HandlerMethod) ? handler.toString() : null;
```

Comment:
What does this actually look like in practice ... some combination of method and supplied parameters?

Reply:
Yes, correct. Here's a real example (from `GET /applications/gate/serverGroups`):

`com.netflix.spinnaker.gate.controllers.ServerGroupController#getServerGroupsForApplication(String, String, String, String, String)`